include/usb/class/usb_cdc.h

```diff
@@ -24,9 +24,6 @@
 /** CDC Specification release number in BCD format */
 #define CDC_SRN_1_20			0x0120
 
-/** CDC Class Code */
-#define COMMUNICATION_DEVICE_CLASS	0x02
```

Comment:
The same here, All definitions for CDC should be here inside usb_cdc.h, if you wont to fix it, then please remove it from common header.